cfgov/v1/jinja2/v1/includes/molecules/call-to-action.html

```diff
@@ -38,8 +38,7 @@ <h2 class="a-heading">
     {% endif %}
 
     {% if value.button %}
```

Comment:
I know you didn't touch this, but this logic could be improved; it should be `{% if value.button.url and value.button.text %}` - `value.button` will be true if any subvalues are set but we only want to render if both are set.